gulpfile.js

```diff
@@ -5,7 +5,7 @@ const del = require('del')
 const fs = require('fs')
 const gulp = require('gulp')
 const gutil = require('gulp-util')
-const sass = require('gulp-sass')
+const sass = require('gulp-sass')(require('sass'))
```

Comment:
Would you please elaborate on why sass was not required before and now is?

Reply:
I will provide a link when I can find it in my browser history, however, this is the official way you have to install gulp-sass

Reply:
https://stackoverflow.com/questions/68417640/gulp-sass-5-does-not-have-a-default-sass-compiler-please-set-one-yourself